plugins/inputs/cisco_telemetry_mdt/cisco_telemetry_mdt.go

```diff
@@ -66,12 +226,96 @@ func (c *CiscoTelemetryMDT) Start(acc telegraf.Accumulator) error {
 		return err
 	}
 
+        //Create Nexus property transform list.
+        c.nxpropMap = make(map[string]int, 100 + len(c.Dmes))
+        c.nxpropMap["asn"] = 2 //check if it's uint64 type then convert to string.
+        c.nxpropMap["subscriptionId"] = 2 //check if it's uint64 type then convert to string.
+	c.nxpropMap["operState"] = 2 //check if it's uint64 type then convert to string.
+
```

Comment:
`c.nxpropMap` should really be a `map[string]func(field *telemetry.TelemetryField, value interface{}) interface{}` and contain function pointers to the conversions instead of this indirections using strings.